cirq-core/cirq/circuits/circuit.py

```diff
@@ -1357,13 +1361,10 @@ def tetris_concat(
 
         Args:
             circuits: The circuits to concatenate.
-            stop_at_first_alignment: Defaults to false. When true, the circuits
-                are never overlapped more than needed to align their starts (in
-                case the left circuit is smaller) or to align their ends (in
-                case the right circuit is smaller). When false, the smaller
-                circuit can be pushed deeper into the larger circuit, past the
-                first time their starts or ends align, until the second time
-                their starts or ends align.
+            align: When to stop when sliding the circuits together.
+                'left': Stop when the starts of the circuits align.
+                'right': Stop when the ends of the circuits align.
+                'first': Stop the first time either the starts or ends align.
```

Comment:
can we add the old explanation here? I think it makes it more understandable.

Reply:
What do you mean? The old explanation is pretty coupled to the argument.

Reply:
Isn't `stop_at_first_alignment = True` is equivalent to `align=FIRST`? Of course I'd rewrite it to cater for the current setup. Something like: 

```
circuits are never overlapped more than needed to align their starts (in case the left circuit is smaller) or to align their ends (in case the right circuit is smaller). 
```